crypto/threads_pthread.c

```diff
@@ -21,30 +26,63 @@ CRYPTO_RWLOCK *CRYPTO_THREAD_lock_new(void)
         OPENSSL_free(lock);
         return NULL;
     }
+# else
+    pthread_mutexattr_t attr;
+
+    CRYPTO_RWLOCK *lock = OPENSSL_zalloc(sizeof(pthread_mutex_t));
```

Comment:
I hate to be finicky, but preferred pattern is to group data declarations, as in
```
    pthread_mutexattr_t attr;;
    CRYPTO_RWLOCK *lock = OPENSSL_zalloc(sizeof(pthread_mutex_t));

   if (lock == NULL)
```
Well, this file doesn't set good example when it comes to empty line after data declaration... So I suppose we can let it slide this time. Empty line after data declarations, but not between them. Sorry.

To be nice to ci, add `[skip ci]` to commit messages for pure reformat commits.

We also reserve the right to squash commits and reformat messages. We can even choose to trivially modify messages without prior notice, though preferred way is to consult with committer.